active-response/firewall-drop.sh

```diff
@@ -32,7 +32,8 @@ IP=$3
 PWD=`pwd`
 LOCK="${PWD}/fw-drop"
 LOCK_PID="${PWD}/fw-drop/pid"
-
+IPV4F="$(cat /proc/sys/net/ipv4/ip_forward)"
```

Comment:
It's not interesting to me, I don't use it. But other people might like it. I have no objections to it. :-)